http://cygwin.com/ml/cygwin-apps/2013-06/msg00109.html

--- origsrc/openldap-2.4.35/libraries/libldap/getattr.c	2013-03-28 10:41:51.000000000 -0500
+++ src/openldap-2.4.35/libraries/libldap/getattr.c	2013-06-12 19:07:31.378388100 -0500
@@ -147,7 +147,7 @@ ldap_get_attribute_ber( LDAP *ld, LDAPMe
 
 		/* skip sequence, snarf attribute type */
 		tag = ber_scanf( ber, vals ? "{mM}" : "{mx}", attr, vals,
-			&siz, 0 ); 
+			&siz, (ber_len_t) 0 );
 		if( tag == LBER_ERROR ) {
 			rc = ld->ld_errno = LDAP_DECODING_ERROR;
 		}
